src/Microsoft.Extensions.Logging.Log4Net.AspNetCore/Log4NetLoggingEventFactory.cs

```diff
@@ -0,0 +1,30 @@
+﻿using log4net.Core;
+using Microsoft.Extensions.Logging.Log4Net.AspNetCore.Entities;
+using System;
+
+namespace Microsoft.Extensions.Logging
+{
+    /// <inheritdoc cref="ILog4NetLoggingEventFactory"/>
+    public class Log4NetLoggingEventFactory
```

Comment:
In my opinion, this should not be public, or at least, should be sealed to avoid inheritance, because the end-consumer should not be able to instantiate default LoggingEventFactory by itself, and in case he/she wants to use its own LoggingEventFactory, should implement it. The exposed type will be always the interface.

Reply:
It's `sealed` now. I thought about making it `internal sealed` but that would require to expose the internals to the test project, so I backed of from that idea for now.